internal/bminventory/inventory.go

```diff
@@ -999,11 +999,15 @@ func (b *bareMetalInventory) InstallCluster(ctx context.Context, params installe
 			log:    log,
 			params: params,
 		}
-		err = b.db.Transaction(cInstaller.install)
-		if err == nil {
-			//send metric when the installation process has been started
-			b.metricApi.InstallationStarted(cluster.OpenshiftVersion, *cluster.ID)
+		if err = b.db.Transaction(cInstaller.install); err != nil {
+			return
 		}
+
+		// send metric and event that installation process has been started
+		b.metricApi.InstallationStarted(cluster.OpenshiftVersion, *cluster.ID)
+		b.eventsHandler.AddEvent(
+			ctx, *cluster.ID, nil, models.EventSeverityInfo,
+			fmt.Sprintf("Updated status of cluster %s to installing", cluster.ID), time.Now())
```

Comment:
Why not use cluster name instead of ID?

Reply:
You're right
If anything, maybe we shouldn't even need the name, as it's already bounded to a specific cluster view

Reply:
No need for name or ID